x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -37,6 +37,7 @@ function getAnomaliesTableData(client: IScopedClusterClient, payload: any) {
     maxRecords,
     maxExamples,
     influencersFilterQuery,
+    actualPlotFunction,
```

Comment:
As before, maybe this should be renamed to `functionDescription` to match the name of the property in the records?

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/cb2429e192079f4e9a1cfc0e401a2a9544d19f23